src/com/severalcircles/colors/commands/Command_about.java

```diff
@@ -0,0 +1,18 @@
+package com.severalcircles.colors.commands;
+
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+public class Command_about implements CommandExecutor {
+
+    @Override
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Annotation 'Override' should be on the same line with its target.](https://app.codacy.com/manual/aidanveney/colors/pullRequest?prid=5519646)